Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM golang:alpine as builder
+FROM golang:1.13-alpine3.10 as builder
```

Comment:
Please change this to `1.13-alpine` only.

Reply:
done